Content.Shared/_RMC14/Attachable/AttachableHolderSystem.cs

```diff
@@ -360,7 +372,7 @@ private void OnDetachDoAfter(EntityUid uid, AttachableHolderComponent component,
 
         if (string.IsNullOrEmpty(slotId))
             CanAttach(holder, attachableUid, ref slotId);
-
+        
```

Comment:
```suggestion

```

Reply:
I have no idea how these got there, but they should be gone now.